Fixed readme (#97)
ZupIT/docs-charles: CONTRIBUTING.md

- Answer the questions about what you did, how to verify it and a description for the changelog, see an example:

## **How to contribute?** 

You can suggest a change, a fix, or improvements to our documentation, such as: 
- Add missing information.
- Fix a grammar or code error.
- Suggest a new section.
- Translate. 

### **Open an issue**
Follow the steps below: 

**Step 1.** Access [**GitHub Issues**]( https://github.com/ZupIT/docs-charles/issues) and open an issue;

**Step 2.** Describe the issue or change you want to make; 

**Step 3.** Make your changes;

**Step 4.** Open a pull request open a PR to the published branch, i. e., **`release-VERSION`** and fill in the template; 

**Step 5.** CharlesCD's team will check your issue, review it, and then approve your PR. 

- Is this the first CharlesCD repository you visit? Check out other [**repositories**](https://github.com/ZupIT/charlescd).

Every time that a new release is created, we save the current work into a new branch, called `release-VERSION`.

## **Code of Conduct**
Please read the [**Code of Conduct**](https://github.com/ZupIT/docs-charles/blob/main/CODE_OF_CONDUCT.md).

## **Community**

> If you have any questions or ideas, let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br).

Reviewer: ```suggestion
Do you have any question about CharlesCD? Let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br).
```
